redwood-widget/src/iosMain/kotlin/app/cash/redwood/widget/RedwoodUIView.kt

```diff
@@ -54,11 +55,14 @@ public open class RedwoodUIView : RedwoodView<UIView> {
   override val children: Widget.Children<UIView>
     get() = _children
 
+  private val density = Density.Default
```

Comment:
```suggestion
  private val density: Density get() = Density.Default
```
No need to burn a backing field on a value that never changes